x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/custom_timeline_data_grid_body.tsx

```diff
@@ -76,12 +80,20 @@ const CustomGridRow = styled.div.attrs<{
   border-bottom: 1px solid ${(props) => (props.theme as EuiTheme).eui.euiBorderThin};
 `;
 
-/* below styles as per : https://eui.elastic.co/#/tabular-content/data-grid-advanced#custom-body-renderer */
+const CustomLazyRowPlaceholder = styled.div.attrs({
+  className: 'customlazyGridRowPlaceholder',
+})`
+  height: 80px;
```

Comment:
Will this have to be configurable once the unified components row height util in the UI is connected?

Reply:
You are right. I forgot about that. I will make it adapt according to row height.